quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -778,19 +777,20 @@ private static void addMappedOptionsToArgGroups(CommandLine commandLine, Map<Opt
                     name = decorateDuplicitOptionName(name);
                 }
 
-                String description = mapper.getDescription();
-
-                if (description == null || cSpec.optionsMap().containsKey(name) || name.endsWith(OPTION_PART_SEPARATOR) || alreadyPresentArgs.contains(name)) {
+                if (cSpec.optionsMap().containsKey(name) || alreadyPresentArgs.contains(name)) {
                     //when key is already added or has no description, don't add.
```

Comment:
```suggestion
                    //when key is already added, don't add.
```

nitpick: might be updated if you potentially make some changes.